docs/traefik/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: '3.2'
+services:
+  traefik:
+    container_name: traefik
+    image: traefik:latest
+    command:
+    - --entrypoints.web.address=:80
+    - --providers.docker=true
+    - --providers.docker.exposedbydefault=false
+    - --api.dashboard=true
+    - --accesslog=true
+    labels:
+    - "traefik.enable=true"
+    - "traefik.http.routers.dashboard.rule=PathPrefix(`/api`) || PathPrefix(`/dashboard`)"
+    - "traefik.http.routers.dashboard.tls=false"
+    ports:
+    - 80:80
+    - 443:443
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock:ro
+  task-store:
+    image: "samply/blaze:0.18"
+    container_name: bridgehead-task-store
+    environment:
+      CONTEXT_PATH: "/task-store"
+      JAVA_TOOL_OPTIONS: "-Xmx1g"
+      LOG_LEVEL: "debug"
+    volumes:
+    - "task-store-data:/app/data"
+    labels:
+    - "traefik.enable=true"
+    - "traefik.http.routers.task_store.rule=PathPrefix(`/task-store`)"
+    - "traefik.http.services.task_store.loadbalancer.server.port=8080"
+  data-store:
+    image: "samply/blaze:0.18"
+    container_name: data-store
+    environment:
+      CONTEXT_PATH: "/data-store"
+      JAVA_TOOL_OPTIONS: "-Xmx1g"
+      LOG_LEVEL: "debug"
+    volumes:
+    - "data-store-data:/app/data"
+    labels:
+    - "traefik.enable=true"
+    - "traefik.http.routers.data_store.rule=PathPrefix(`/data-store`)"
+    - "traefik.http.services.data_store.loadbalancer.server.port=8080"
+  report-hub:
+    image: "samply/report-hub:latest"
```

Comment:
This will not work for branch, since the build is only for main. Please consider this for the future.